client/src/components/Users/Auth/SignUpCitizen/apis/backend.ts

```diff
@@ -27,6 +27,12 @@ const Endpoints = {
   checkUserEmail: (userEmail: string) => {
     return client.get(`/auth/user-email-exists/${userEmail}`);
   },
+  userUpdateProfile: (profile: Profile) => {
+    const { file, userId } = profile;
+    const formData = new FormData();
+    formData.append('profile_image', file);
```

Comment:
Could we also use `formData.set` here?  (just for my own understanding, not a request to change)

Reply:
from Mozilla docs:
>The difference between [set()](https://developer.mozilla.org/en-US/docs/Web/API/FormData/set) and append() is that if the specified key already exists, set() will overwrite all existing values with the new one, whereas append() will append the new value onto the end of the existing set of values.